network_analysis/find-service-areas-for-multiple-facilities/build.gradle

```diff
@@ -67,9 +67,9 @@ task productionZip(type: Zip) {
         into "samples-data"
     }
     from copyNatives
-    from jar.destinationDir
+    from jar.destinationDirectory
```

Comment:
the gradle build check seems to be throwing a build failure here

```
Caused by: groovy.lang.MissingPropertyException: Could not get unknown property 'destinationDirectory' for task ':feature-collection-layer:jar' of type org.gradle.api.tasks.bundling.Jar.
	at org.gradle.internal.metaobject.AbstractDynamicObject.getMissingProperty(AbstractDynamicObject.java:83)
	at org.gradle.internal.metaobject.AbstractDynamicObject.getProperty(AbstractDynamicObject.java:61)
	at org.gradle.api.tasks.bundling.Jar_Decorated.getProperty(Unknown Source)
```

Reply:
@sclaridge I found the culprit! Thanks to Jon's thoughts on why things were going awry a while back, we need to upgrade the highest level gradle-wrapper.properties file to ensure it references 6.5.1 and not 5.0.

Reply:
Thanks for noticing this, the build is successful now!